File: src/main/java/ee/leola/kassa/repository/UserRepository.java
package ee.leola.kassa.repository;

import ee.leola.kassa.models.User;
import java.math.BigDecimal;
import java.util.List;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface UserRepository extends JpaRepository<User, Long> {
  List<User> findByBalanceNot(BigDecimal balance);
}

File: src/main/kotlin/ee/leola/kassa/repository/UserRepository.kt
package ee.leola.kassa.repository

import ee.leola.kassa.models.User
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.math.BigDecimal

@Repository
interface UserRepository : JpaRepository<User, Long> {
    fun findByBalanceNot(balance: BigDecimal): List<User>
}
